native/python/pyjp_value.cpp

```diff
@@ -43,8 +45,28 @@ PyObject* PyJPValue_alloc(PyTypeObject* type, Py_ssize_t nitems )
 	JP_PY_TRY("PyJPValue_alloc");
 	// Modification from Python to add size elements
 	const size_t size = _PyObject_VAR_SIZE(type, nitems + 1) + sizeof (JPValue);
```

Comment:
Use of ``_PyObject_VAR_SIZE`` here looks like it could be a future candidate for breaking change. (cc @vstinner in light of the comment in https://bugs.python.org/msg411999)

Reply:
There was nothing pulbic with tge same functionality.  We could copy it into our code if needed.